### PR TITLE
v0.6 release news post

### PR DESCRIPTION
**Pull request by hashhar**
screenshot 263
screenshot 264

### CODE REVIEW

**Comment by kathweaver** [comment]
Did you put the two download files in?  Anytime ###download is in a news post it disappears.

**Comment by hashhar** [comment]
The ones in [this](https://files.gitter.im/OpenLiveWriter/OpenLiveWriter/W9CA/Screenshot-from-2016-02-13-01-14-43.png)? Nope. They are part of an older pull.

**Comment by kathweaver** [comment]
If you look at his release notes, there are two files referenced at the bottom. 

If you look at the old news files, the files are referenced but the header is missing.

**Comment by hashhar** [comment]
Yup. Those are links to the source code. As soon as #57 is merged they will automatically be visible at the bottom of relevant pages. The download for the setup is accessible through the big blue button.

**Comment by kathweaver** [comment]
They look strange on the main page.

**Comment by ScottIsAFool** [comment]
*cough* where's my name? *cough* :)

**Comment by hashhar** [comment]
Oooops. Fixed.

File: _posts/2016-02-14-version-0-dot-6-now-available.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+After about two months since the initial public release the team and community have managed to add features, fix issues and fulfill the demand for some of the more requested features like spell checking (it will only work on Windows 8 and later though) and support for Blogger categories. We hope you enjoy the new features as much as we enjoyed working on them. Read on to learn more about what issues have been fixed and what features have been added.
+
+The next time you start Open Live Writer and are connected to the internet, it should get updated automatically. But if you can't wait to try out the goodness, download the [updated setup here](https://openlivewriter.azureedge.net/stable/Releases/OpenLiveWriterSetup.exe).
+
+## Release Notes
+
+For spell checking to work you'll need to have Windows 8 or newer installed (Windows 7 does not have spell checking support) and you’ll need to install your preferred input language in Windows to enable spell checking in that language. Only one language can be spell checked in Open Live Writer at a time, but you can change the language at any time by going to **File > Options > Spelling** and choosing a new dictionary language.
```
Comment (ScottIsAFool) [inline _posts/2016-02-14-version-0-dot-6-now-available.md]: Can we get a link to the instructions on how to install the input language. There should be a link on the release notes. We should definitely have this mentioned in here too.

**Comment by kathweaver** [comment]
Where are the instructions for how to install the input language.  We should link tutorials too.

**Comment by ScottIsAFool** [comment]
Here http://windows.microsoft.com/en-us/windows-10/how-to-add-an-input-language-to-your-pc#v1h=tab01. Will put a link to it in the release on the other github repository :)

**Comment by kathweaver** [comment]
Okay, and I’ll write a tutorial for it and a few other things for our website.

**Comment by ScottIsAFool** [comment]
@hashhar Can we get the link above mentioned in the blog post please, same as per release notes on the main repo. Once that's added, I'll merge in.

**Comment by hashhar** [comment]
@ScottIsAFool Added.

**Pull request merged**